### PR TITLE
[5.5] Hotfix/cyrillic ascii map

### PR DESCRIPTION
**Pull request by iipavlov**
Fixing the cyrillic map. Most notably `ь` and `ъ` where incorrect transliterated to `b`.  
Used the mapping for [BGN/PCGN romanization of Bulgarian - BGN/PCGN 2013 system reflecting the official Bulgarian system](https://en.wikipedia.org/wiki/Romanization_of_Bulgarian#Streamlined_System_with_-ia-exception).
As exception the transliteration of `щ/Щ` to `shch/SHCH` was kept for compatibility with the Russion sound for this character.

### CODE REVIEW

**Comment by browner12** [comment]
please don't commit `.gitignore` changes with this.

sidenote. git has a global gitignore file that you can throw '.idea' into so you never have to worry about it on these open source projects.

https://help.github.com/articles/ignoring-files/

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,5 @@ composer.lock
 .php_cs.cache
 .DS_Store
 Thumbs.db
-/phpunit.xml
\ No newline at end of file
+/phpunit.xml
+.idea/
```
Comment (tillkruss) [inline .gitignore]: Why are you adding this?

**Comment by taylorotwell** [comment]
Changes .gitignore which is irrelevant. Also, this map should basically just be kept in sync with Stringy. Is that the case here?

**Comment by iipavlov** [comment]
Sorry about the .gitignore - swept out by the style corrections. @browner12 , Thanks for the advice!

@taylorotwell, sorry, I was not aware of the requirement to be kept in sync with Stringly. Is danielstjules/Stringy the correct repository ?
I just tried to make the Laravel transliteration closer to a common standard. however there are multiple such, so probably simpler would be to use an inherited custom class.
Anyway transliterating `ъ` with  `b`is not a good match - the first is vowel, pronounced as in 'cut.

**Comment by fernandobandeira** [comment]
@iipavlov That's the correct repo, you can create a PR there and we'll sync it when that repo releases a new version 😄

**Pull request closed without merging**